### PR TITLE
feat: add variable for aliyun theme of divider

### PR DESCRIPTION
**Pull request by hi-zhaoyeqing**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [x] Other (add variable for aliyun theme of divider)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |feat: add variable for aliyun theme of divider|
| 🇨🇳 Chinese |新功能：为divider组件添加阿里云主题变量|

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

**Review by vthinkxie — APPROVED** [review]
LGTM

File: components/divider/style/index.less [hunk 1]
```diff
@@ -13,7 +13,7 @@
     top: -0.06em;
     display: inline-block;
     height: 0.9em;
-    margin: 0 8px;
+    margin: 0 @divider-horizontal-margin;
```
Comment (afc163) [inline components/divider/style/index.less:16]:
```suggestion
    margin: 0 @divider-margin-horizontal;
```

File: components/style/themes/variable.less [hunk 2]
```diff
@@ -338,6 +338,7 @@ html {
 @divider-text-padding: 1em;
 @divider-orientation-margin: 5%;
 @divider-color: rgba(0, 0, 0, 6%);
+@divider-horizontal-margin: 8px;
```
Comment (afc163) [inline components/style/themes/variable.less:341]: @zombieJ 现在需要两个文件都写一遍么？

**Comment by qdzhaoxiaodao** [comment]
本次提交的pr是为了对齐xconsle阿里云主题的divider新增的变量，可以控制左右的margin
image

File: components/divider/style/index.less (hunk 1 above)
Comment (qdzhaoxiaodao) [reply components/divider/style/index.less]: 已修改成@divider-margin-horizontal

File: components/divider/style/index.less [hunk 3]
```diff
@@ -13,7 +13,7 @@
     top: -0.06em;
     display: inline-block;
     height: 0.9em;
-    margin: 0 8px;
+    margin: 0 @divider-margin-horizontal;
```
Comment (afc163) [inline components/divider/style/index.less:16]: 哦，这是垂直的分割线。叫 `@divider-vertical-gutter` 更合适一点。

Comment (qdzhaoxiaodao) [reply components/divider/style/index.less]: 已修改

**Review by afc163 — APPROVED** [review]

**Review by zombieJ — APPROVED** [review]

**Pull request merged**